src/calculateRentalCost.js

```diff
@@ -3,8 +3,19 @@
  *
  * @return {number}
  */
+const discountsDays1 = 3;
+const discountsDays2 = 7;
+const discount1 = 20;
+const discount2 = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days < discountsDays1) {
+    return days * 40;
+  } else if (days >= discountsDays1 && days < discountsDays2) {
+    return days * 40 - discount1;
+  } else if (days >= discountsDays2) {
+    return days * 40 - discount2;
+  }
```

Comment:
Instead of using `else if` and `else`, prefer using multiple `if` statements with `return`. This simplifies the logic and makes it easier to follow. Also, it's a good practice to avoid unnecessary `else` after an `if` that contains a `return`.